src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { useTodo } from '../../hooks/useTodo';
+import { Todo } from '../../types/Todo';
+import './animations.scss';
+
+type Props = {
+  tempTodo: Todo | null;
+};
+
+export const TodoList: React.FC<Props> = ({ tempTodo }) => {
+  const { visibleTodos } = useTodo();
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings